pkg/apis/componentconfig/types.go

```diff
@@ -62,3 +62,213 @@ const (
 	ProxyModeUserspace ProxyMode = "userspace"
 	ProxyModeIPTables  ProxyMode = "iptables"
 )
+
+type KubeletConfiguration struct {
+	// config is the path to the config file or directory of files
+	Config string `json:"config"`
```

Comment:
Agreed alpha ordering isn't a pure win, but I think it is an improvement over this.  docker --help and kubectl --help both show flags in alphabetical ordering as do most man pages, so it seems like a reasonable convention for this struct of flags.  If you could change the code generator to sort alphabetically and I'd prefer to standardize on it.